buildenv/jenkins/jobs/infrastructure/Cleanup-Nodes.groovy

```diff
@@ -184,7 +184,7 @@ timeout(time: TIMEOUT_TIME.toInteger(), unit: TIMEOUT_UNITS) {
 
                                     // Cleanup zOS datasets 
                                     if (nodeLabels.contains('sw.os.zos')) {
-                                        listcat = sh(script: "tso listcat | grep '${env.USER}' | cut -d. -f 2-", returnStdout: true).trim()
+                                        listcat = sh(script: "tso listcat | grep '${env.USER}' | grep 'JVM' | cut -d. -f 2-", returnStdout: true).trim()
```

Comment:
Do all the dataset names of interest start with "JVM"? If so, the second grep pattern should be `'^JVM'`.

Reply:
@keithc-ca , datasets of interest starts with 'JENKINS.JVM' like JENKINS.JVM.JENKINS2.D240506.T100125.X001

Reply:
Ok, then I suggest that we should only match `.JVM.`, i.e. with `grep -F .JVM.`.

Reply:
@keithc-ca ,
Not all removable datasets have  ".JVM". in them some have ".MYJVM."

datasets in zos jenkins machines: https://hyc-runtimes-jenkins.swg-devops.com/job/all-nodes-matching-labels/1535/console
```
01:14:15  JENKINS.JVM.JENKINS9.D240513.T181148.X001
01:14:15  JENKINS.JVM.JENKINS9.D240514.T093028.X001
01:14:15  JENKINS.MYJVM.JENKINS4.D240514.T080530.X001
01:14:15  JENKINS.MYJVM.JENKINS6.D240514.T080544.X001
```

Reply:
Perhaps the pattern is as good as we can hope for, but I worry we'll delete datasets that should have been kept if the pattern is not specific enough.